README.md

```diff
@@ -16,6 +16,17 @@ Deis v2 and Helm are changing quickly. Your feedback and participation are more
 
 Hacking on Deis v2 is a bit rough right now--please help us fix the bugs and improve the installation process.
 
+### Prerequisites
+You must enable the extensions api for your kube-apiserver. For now the only requirement is that the daemonsets portion of the api be enabled. To do this add the following line to the script that starts your api server:
+
+```console
+--runtime-config=extensions/v1beta1=true,extensions/v1beta1/daemonsets=true
+```
+
+you can also do `ENABLE_DAEMONSETS=true` before running the `kube-up.sh` script.
+
+### Installation 
+
 First, add this Chart repo to Helm to install the "deis" chart:
```

Comment:
I feel like this section is making some pretty significant assumptions about _how_ one has gotten from "zero to kubernetes."  Even on our own team, we have at least half a dozen ways of provisioning k8s, with `kube-up` being just one of them.  This is actually a good thing because it's prepping us for a world of k8s ubiquity where not every cluster has been provisioned through the same means... but imo, it means we also need to be sensitive to that in the docs.

Maybe this can at least be qualified with "if you're using `kube-up`... otherwise, consult your Kubernetes distribution's documentation?"